addons/l10n_es/data/account_account_template_common_data.xml

```diff
@@ -1,4143 +1,4121 @@
             <field name="chart_template_id" ref="l10n_es.account_chart_template_common"/>
         </record>
-        <record id="pgc_2903_child" model="account.account.template">
+        <record id="l10n_es.account_common_2903" model="account.account.template">
             <field name="code">2903</field>
             <field name="reconcile" eval="False"/>
             <field name="name">Deterioro del valor de propiedad industrial</field>
             <field name="user_type_id" ref="account.data_account_type_fixed_assets"/>
             <field name="chart_template_id" ref="l10n_es.account_chart_template_common"/>
         </record>
-        <record id="pgc_2905_child" model="account.account.template">
+        <record id="l10n_es.account_common_2905" model="account.account.template">
             <field name="code">2905</field>
             <field name="reconcile" eval="False"/>
             <field name="name">Deterioro del valor de derechos de traspaso</field>
             <field name="user_type_id" ref="account.data_account_type_fixed_assets"/>
             <field name="chart_template_id" ref="l10n_es.account_chart_template_common"/>
         </record>
-        <record id="pgc_2906_child" model="account.account.template">
+        <record id="l10n_es.account_common_2906" model="account.account.template">
             <field name="code">2906</field>
             <field name="reconcile" eval="False"/>
             <field name="name">Deterioro del valor de aplicaciones informáticas</field>
             <field name="user_type_id" ref="account.data_account_type_fixed_assets"/>
             <field name="chart_template_id" ref="l10n_es.account_chart_template_common"/>
         </record>
-        <record id="pgc_2910_child" model="account.account.template">
+        <record id="l10n_es.account_common_2910" model="account.account.template">
             <field name="code">2910</field>
             <field name="reconcile" eval="False"/>
             <field name="name">Deterioro de valor de terrenos y bienes naturales</field>
             <field name="user_type_id" ref="account.data_account_type_fixed_assets"/>
             <field name="chart_template_id" ref="l10n_es.account_chart_template_common"/>
         </record>
-        <record id="pgc_2911_child" model="account.account.template">
+        <record id="l10n_es.account_common_2911" model="account.account.template">
             <field name="code">2911</field>
             <field name="reconcile" eval="False"/>
             <field name="name">Deterioro de valor de construcciones</field>
             <field name="user_type_id" ref="account.data_account_type_fixed_assets"/>
             <field name="chart_template_id" ref="l10n_es.account_chart_template_common"/>
         </record>
-        <record id="pgc_2912_child" model="account.account.template">
+        <record id="l10n_es.account_common_2912" model="account.account.template">
        
```

Comment:
Why removing these accounts?

Reply:
They haven't been removed, but renamed, for clarity and soundness with the "new" accounts defined for PYMES, "assoc" and "full" CoA. Naming them so allows knowing where they come from with just their xml id.